nevergrad/optimization/test_doc.py

```diff
@@ -45,15 +45,19 @@ def square(x, y=12):
     with futures.ThreadPoolExecutor(max_workers=optimizer.num_workers) as executor:
         recommendation = optimizer.minimize(square, executor=executor, batch_mode=False)
     # DOC_BASE_3
-    optimizer = ng.optimizers.OnePlusOne(parametrization=instrum, budget=10, num_workers=1)
+    import nevergrad as ng
+    def square(x, y=12):
+        return sum((x - .5)**2) + abs(y)
+    instrum = ng.p.Instrumentation(ng.p.Array(shape=(2,)), y=ng.p.Scalar())  # We are working on R^2.
```

Comment:
the comment is wrong (probably to remove somewhere else then), this is R2xR

Reply:
Done, thx.